### DISCUSSION TITLE
Augmentation level

### DISCUSSION THREAD

**Discussion by alexeybozhchenko** [open-mmlab/mmdetection#9116] [Q&A] [answered]
Greeting. What does the level argument in auto_augment.Rotate? from the code I suggest it just a multiplier that adjusts given max_angle argument, but how do I interpret it precisely? For example, by adding
`dict(type='Rotate', level=10, max_rotate_angle=40)` I expect the pipeline to augment my training data by rotating a sample(including mask and bboxes) for a maximum of 40 degrees with a default proba of 0.5, is that correct? Also may be there is a feature that allows me to inspect my augmentations? Thank you

**Comment by ZwwWayne** [comment, marked as answer]
Yes. You understanding is correct. You can use browse_dataset.py in tools to do that.
